addOns/llm/src/main/java/org/zaproxy/addon/llm/services/LlmAssistant.java

```diff
@@ -1,34 +1,56 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.zaproxy.addon.llm.services;
 
 import dev.langchain4j.service.SystemMessage;
 import dev.langchain4j.service.UserMessage;
-
 import dev.langchain4j.service.V;
 import org.zaproxy.addon.llm.communication.Confidence;
 import org.zaproxy.addon.llm.communication.HttpRequestList;
 
 public interface LlmAssistant {
-    @UserMessage("Given the following swagger generate list of chained HTTP request to simulate a real world user : {{swagger}} ")
+    @UserMessage(
+            "Given the following swagger generate list of chained HTTP request to simulate a real world user : {{swagger}} ")
     HttpRequestList extractHttpRequests(String swagger);
 
-    @UserMessage("As software architect, and based on your previous answer, generate other potential missing endpoint that are not mentioned on the swagger file. For example, if there is GET /product/1, suggest DELETE /product/1 if it's not mentioned")
+    @UserMessage(
+            "As software architect, and based on your previous answer, generate other potential missing endpoints that are not mentioned in the swagger file. For example, if there is GET /product/1, suggest DELETE /product/1 if it's not mentioned")
     HttpRequestList complete();
 
-    @SystemMessage("You are a web application security expert in review false positives. Answer only in JSON.")
-    @UserMessage("Your task is to review the following finding from ZAP (Zed Attack Proxy).\n"
-            + "The confidence level is a pull down field which allows you to specify how confident you are in the validity of the finding : \n"
-            + "- 0 if it's False Positive\n"
-            + "- 1 if it's Low\n"
-            + "- 2 if it's Medium\n"
-            + "- 3 if it's High\n"
-            + "- 4 if it's Confirmed\n"
-            + "\n"
-            + "The alert is described as follow : {{description}}\n"
-            + "\n"
-            + "As evidence, the HTTP response contains :\n"
-            + "---\n"
-            + "{{evidence}}\n"
-            + "---\n"
-            + "Provide short consistent explanation of the new score.\n")
+    @SystemMessage(
+            "You are a web application security expert in review false positives. Answer only in JSON.")
+    @UserMessage(
+            "Your task is to review the following finding from ZAP (Zed Attack Proxy).\n"
+                    + "The confidence level is a pull down field which allows you to specify how confident you are in the validity of the finding : \n"
+                    + "- 0 if it's False Positive\n"
+                    + "- 1 if it's Low\n"
+                    + "- 2 if it's Medium\n"
+                    + "- 3 if it's High\n"
+                    + "- 4 if it's Confirmed\n"
```

Comment:
Would there be any benefit to describing or noting that Confirmed is only ever user set? It is never set my scan rules or ZAP itself.